spec/queue.js

```diff
@@ -0,0 +1,65 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Queue from '../src/queue'
+
+chai.use(chaiChange)
+
+describe.only('Queue', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(Queue).to.be.a('function')
+  })
+
+  context('enqueue()', () => {
+    it('adds an element to the back of the queue', () => {
+      const myQueue = new Queue()
+      myQueue.enqueue('foo')
+      myQueue.enqueue('bar')
+      expect(myQueue.elements[1]).to.eql('bar')
+    })
+  })
+
+  context('dequeue()', () => {
+    it('adds an element to the back of the queue', () => {
+      const myQueue = new Queue()
+      myQueue.enqueue('foo')
+      myQueue.enqueue('bar')
+      expect(myQueue.dequeue()).to.eql(['foo'])
```

Comment:
Should dequeue return the element or the element in an array,
I would think that it would be
```javascript
expect(myQueue.dequeue()).to.eql('foo')
```